src/index.js

```diff
@@ -128,19 +135,11 @@ const ApolloClientPlugin: FusionPlugin<
         : [authMiddleware, connectionLink];
 
       const client = new ApolloClient({
-        // ssrMode must be set to true in order to use SSR hydrated cache.
-        ssrMode: true,
+        ssrMode: __NODE__,
```

Comment:
Does this cause queries to refetch on the browser, or does the undefined defaultOptions value fix that?

Reply:
The undefined defaultOptions fixes this. We were previously overwriting the `query` fetchPolicy to be `cache-and-network`.